app/storyedit/directives/hero.component.ts

```diff
@@ -18,18 +20,18 @@ import {SpinnerComponent}  from '../../shared/spinner/spinner.component';
         <spinner *ngIf="!story.isLoaded" inverse=true></spinner>
         <div class="info" *ngIf="story.isLoaded">
           <h2>{{story.title || '(Untitled)'}}</h2>
-          <p *ngIf="story.modifiedAt">Last saved at {{story.modifiedAt | date}}</p>
+          <p *ngIf="story.updatedAt">Last saved at {{story.updatedAt | timeago}}</p>
         </div>
         <div class="actions">
           <button class="preview">Preview</button>
-          <button *ngIf="!story.id" class="create"
-            [disabled]="!story.isValid"
+          <button *ngIf="story.isNew" class="create"
+            [disabled]="story.invalid()"
             (click)="save()">Create</button>
-          <button *ngIf="story.id" class="save"
-            [disabled]="!story.isValid || !story.isChanged"
+          <button *ngIf="!story.isNew" class="save"
+            [disabled]="story.invalid() || !story.changed()"
```

Comment:
Yeah, these model-style bindings seem to work pretty well.  Couple of UX issues to address down the road (how/when to show changed/invalid states).  But the bindings are all there if we need them.